.gitmodules

```diff
@@ -2,3 +2,6 @@
   path = sorc/ncep_post.fd/post_gtg.fd
   url = https://github.com/NCAR/UPP_GTG
   update = none
+[submodule "sorc/libIFI.fd"]
+	path = sorc/libIFI.fd
+	url = ssh://git@github.com/NCAR/UPP_IFI
```

Comment:
@SamuelTrahanNOAA Could this url link be changed to the https link? Could you add a line "update = none" (see line 4) so the general users from ufs-weather-model might not be bothered by the error message of no access to UPP_IFI repository when they recursively check out the whole ufs-weather-model package? We implemented it for skipping UPP_GTG checkout.  The general users will get a message for skipping "sorc/ncep_post.fd/post_gtg.fd". The gtg authorized users can check out the code as: 
```
git -c submodule."post_gtg.fd".update=checkout submodule update --init --recursive
```
Please let me know your thoughts.

Reply:
Done.